File: Data Structures & Algorithms/single-number-iii/submission-0.cpp
class Solution {
public:
    vector<int> singleNumber(vector<int>& nums) {
        unordered_map<int, int> count;
        for (int num : nums) {
            count[num]++;
        }

        vector<int> res;
        for (const auto& pair : count) {
            if (pair.second == 1) {
                res.push_back(pair.first);
            }
        }

        return res;
    }
};

File: Data Structures & Algorithms/find-duplicate-integer/submission-0.java
class Solution {
    public int findDuplicate(int[] nums) {
        for (int i = 0; i < nums.length; i++) {
            for (int k = i + 1; k < nums.length; k++) {
                if (nums[i] == nums[k]) {
                    return nums[k];
                }
            }
        }
        return nums[0];
    }
}
